### PR TITLE
Add doc for scalacheck module

### PR DESCRIPTION
**Pull request by road21**
+ some minor fixes in "test" of arbitrary deriving

### CODE REVIEW

**Comment by REDNBLACK** [comment]
@road21 could you please update pkg name in tests?

**Comment by road21** [comment]
> @road21 could you please update pkg name in tests?
updated

**Comment by road21** [comment]
> @road21 could you please update pkg name in tests?

updated

File: README.md [hunk 1]
```diff
@@ -202,4 +202,28 @@ import derevo.derive
...
 // TODO
+```
+
+### [Scalacheck](https://github.com/typelevel/scalacheck)
+```sbt
+libraryDependencies += "org.manatki" %% "derevo-scalacheck" % "latest version in badge" % Test
+```
+
+```scala
+import org.manatki.derevo.derive
```
Comment (REDNBLACK) [inline README.md]: old pkg name needs to be updated

Comment (road21) [reply README.md]: uops, sorry, fixed

**Pull request merged**